src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -93,8 +112,17 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
     projectNumber: changeRequest.wbsNum.projectNumber,
     workPackageNumber: 0
   });
+  const [reviewerIds, setReviewerIds] = useState<number[]>([]);
+  const { data: users, isLoading: isLoadingAllUsers, isError: isErrorAllUsers, error: errorAllUsers } = useAllUsers();
+  const requestCRReview = useRequestCRReview(changeRequest.crId.toString());
```

Comment:
this is weird, do it like this intead: 
```suggestion
  const { mutateAsync: requestCRReview } = useRequestCRReview(changeRequest.crId.toString());
```